<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThrowDice</class>
 <widget class="QWidget" name="ThrowDice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Throw Dice</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QSpinBox" name="maxSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="throwDiceBtn">
     <property name="text">
      <string>Throw</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Engine/EResources.qrc">
       <normaloff>:/images/dice</normaloff>:/images/dice</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="minSpinBox">
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Maximum value:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Minimum value:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="infoLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="countSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dice count:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>throwDiceBtn</tabstop>
  <tabstop>minSpinBox</tabstop>
  <tabstop>maxSpinBox</tabstop>
  <tabstop>countSpinBox</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Engine/EResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>ThrowDice</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
